app/views/schools/meters/index.html.erb

```diff
@@ -1,7 +1,10 @@
 <h1><%= t('schools.meters.index.title') %></h1>
 
 <% if current_user.admin? %>
-<div class="mb-2 alert alert-secondary">
+  <% if can? :manage, DashboardMessage %>
+    <%= render 'admin/shared/dashboard_message', messageable: @school %>
+  <% end %>
+<div class="alert alert-secondary mb-2">
   <%= link_to t('schools.meters.index.school_downloads'), school_downloads_path(@school), class: 'btn' %>
```

Comment:
This looks a bit sus. Think the link_to should be inside the following?
<% if can?(:validate_meters, @school) && @school.meters_with_readings.any? %>
<% end %>